### PR TITLE
Provide enums for DT_FLAGS and DT_FLAGS_1

### PR DESCRIPTION
**Pull request by rupran**
This change adds two enums with the name to value mappings
for the two flags fields in the dynamic section. The values
and corresponding names are taken from the elf/elf.h file
in the most recent glibc version.

The enums are also used to print the names instead of the
raw hex values for DT_FLAGS and DT_FLAGS_1 in
scripts/readelf.py.

Fixes: #189

### CODE REVIEW

File: scripts/readelf.py [hunk 1]
```diff
@@ -425,6 +426,10 @@ def display_dynamic_tags(self):
                     parsed = 'Library soname: [%s]' % tag.soname
                 elif tag.entry.d_tag.endswith(('SZ', 'ENT')):
                     parsed = '%i (bytes)' % tag['d_val']
+                elif tag.entry.d_tag == 'DT_FLAGS':
+                    parsed = describe_dt_flags(tag.entry.d_val)
```
Comment (eliben) [inline scripts/readelf.py]: No "Flags: " needed here?

**Comment by eliben** [comment]
Thanks for the contribution! It'd be great to have a test here

File: scripts/readelf.py (hunk 1 above)
Comment (rupran) [reply scripts/readelf.py]: That's what my system's `readelf` prints, so I figured I'd add it here as well.

**Comment by rupran** [comment]
I can add a test case, that would mean adding a binary with some of the flags enabled, right?

File: scripts/readelf.py (hunk 1 above)
Comment (rupran) [reply scripts/readelf.py]: Ah, sorry, I just realized I read your comment wrong. For DT_FLAGS, `readelf` only shows the respective flag names, while for DT_FLAGS_1, the additional "Flags: " is printed... This is also the case in the `readelf` binary shipped with the tests.

**Pull request merged**